conf/fe.conf

```diff
@@ -38,7 +38,7 @@ JAVA_OPTS_FOR_JDK_9="-Xmx4096m -XX:SurvivorRatio=8 -XX:MaxTenuringThreshold=7 -X
 sys_log_level = INFO
 
 # store metadata, create it if it is not exist.
-meta_dir = ${DORIS_HOME}/palo-meta
```

Comment:
I am afraid this may cause some problem.
Some users may not notice this configuration and use the default `palo-meta` directory directly. When the cluster is upgraded, if it is replaced with the `doris-meta` directory, some problems may occur. This directory configuration recommendation remains unchanged.

Reply:
Hi @morningman, for this change I don't agree with you.

1. configuration files will not be updated in normal case.
2. Even if he uses the new configure file, the FE will exit with error messages like "There is no meta directory". And this can be fixed by change this option to the correct directory.

In my opinion, code must keep compatible with old versions. However, configuration default value is unnecessary.

Reply:
How about change this to 

```
# meta_dir = ${DORIS_HOME}/doris-meta
```

So that it is same as default value in `be.conf`, which is also commented out.

And we should also change this in FE's `config.java`

Reply:
It works for me.
Comment for this configuration should be recorded in this file carefully. And related documents and tutorials should be changed too.

Reply:
OK. I will change meta_dir and storage_root_path commented out.

Reply:
OK. I will make meta_dir and storage_root_path commented out, and leave the default value changed to doris.